src/Sylius/Bundle/FlowBundle/Process/Builder/ProcessBuilder.php

```diff
@@ -79,6 +79,10 @@ public function add($name, $step)
             throw new \InvalidArgumentException('Step added via builder must implement "Sylius\Bundle\FlowBundle\Process\Step\StepInterface"');
         }
 
+        if (false === $step->isActive()) {
```

Comment:
It's not a big deal here, we got a contract that `isActive()` returns `bool` so either `false === isActive()` or `!isActive()` are all right and we're just wasting our time on this discussion which depends on personal preferences.

IMO I'd go for `!isActive()` just for one reason - to make a distinction between functions returning mixed values (like `strpos()` - `bool|int`), where `!strpos()` will be `true` if `strpos() === 0 ` and those which are restricted to return `bool` only.